contracts/SnappAuction.sol

```diff
@@ -241,6 +245,34 @@ contract SnappAuction is SnappBase {
         emit AuctionSettlement(slot, stateIndex(), _newStateRoot, pricesAndVolumes);
     }
 
+    function calculateOrderHash(uint slot, uint128[] memory _standingOrderIndex)
+    public view returns (bytes32) {
+        bytes memory batchHashSequence = abi.encode(auctions[slot].shaHash);
```

Comment:
Should this be `encodePacked`? Why or why not?

Reply:
It will not make a difference... on bytes32, they are operating the same

Reply:
You are right. I'd still prefer to stick to use the same in all of our codebase.

Reply:
nit: maybe it's simpler if we extract all orderHashes into an array of bytes32 and only pack them once?